packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -2249,6 +2249,9 @@
   },
   "observability-onboarding-state": {
```

Comment:
Might be useful to rename this `observability-onboarding-state` to `observability-onboarding-flow` to match the new pattern for routes.

Reply:
I tried to rename the savedObject but got the following error
`Removing mapped properties is disallowed >8.8`

Reply:
ah ok...well it's more of a cosmetic thing i guess. i figured now would be the time since it's technically still not released yet.

Reply:
It's not released to serverless yet but I believe modifying the name of a saved object will trigger a more expensive migration that affects customers. This is mainly a problem for heavy users of cases and/or alerts. CC @gsoldevila 

That being said, I think it'd be good to get some "practice" of what it'd be like to be running a live service so I agree  with the decision to rather forgo the cosmetic change :)